src/makeCalculator.js

```diff
@@ -37,7 +37,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
+
+    divide(a, b) {
+      if (b === 0) {
+        throw new Error('Can\'t divide by zero');
```

Comment:
better use interpolation
```suggestion
        throw new Error(`Can't divide by zero`);
```